src/views/shared/base.njk

```diff
@@ -83,6 +83,7 @@
                    {% if errorState %}
                         {{ govukErrorSummary({
                             titleText: errorTitle | default('Error Summary'),
+                            attributes: { autoFocus: true, tabindex: 0 },
```

Comment:
It looks as though the `govukErrorSummary` should automatically focus the element for us - do you know why this isn't working? Code is https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/error-summary/error-summary.mjs but I would sort of expect this to 'just work'...

I wonder if we're not properly loading the GOV.UK javascript :/

Reply:
see better fix for this https://github.com/govuk-one-login/ipv-core-front/pull/1398